src/components/event-details/EventDetails.tsx

```diff
@@ -81,17 +81,27 @@ const EventDetails: React.FC<EventDetailsProps> = ({
 
   const { description, location, start, end } = details;
 
-  const computeDateString = (startDate: Date) => {
-    const fullDate = dateFormat(startDate, 'fullDate');
-
-    return fullDate;
-  };
-
-  const computeTimeString = (startDate: Date, endDate: Date) => {
-    const startTime = dateFormat(startDate, 'shortTime');
-    const endTime = dateFormat(endDate, 'shortTime');
-
-    return `${startTime} - ${endTime}`;
+  const computeDateString = () => {
+    const startDate = dateFormat(start, 'longDate');
+    const endDate = dateFormat(end, 'longDate');
+    const startTime = dateFormat(start, 'shortTime');
+    const endTime = dateFormat(end, 'shortTime');
+    if (startDate === endDate) {
```

Comment:
Could also do a ternary operator here to avoid the if/else statements.

```
startDate === endDate ? (
  <>
     ...
  </>
) : (
  <>
    ...
  </>
}
```

Reply:
good call, thats cleaner